garageScript/c0d3-app
# Create editExerciseComment mutation and update some snapshots for tests

**Description:** relates to https://github.com/garageScript/c0d3-app/issues/2400

This PR adds a graphQL mutation that will handle editing of exercise comments as part of the discussions feature in C0D3 Dojo.
To test:
Use an existing exerciseComment `id` or create a new one in graphQL like so:

1. Go to 'api/graphql'
2. write a query as follows:
```
mutation AddExerciseComment{   
       addExerciseComment(exerciseId:1, content:"Cool")
             { id 
              exerciseId 
              content   } 
}
```

3. You should receive a response such as: 
```
{   "data": {
        "addExerciseComment": { 
            "id": 3, 
            "exerciseId": 1, 
            "content": "Cool" }   
                  }
 }
```
![image](https://user-images.githubusercontent.com/77421872/215617758-eb989716-ffb0-4e33-ab49-456bac81a67e.png)

4. Remember/save the `id` that is generated/assigned to the newly created `exerciseComment`

5. Use this id to run a query such as:
```
mutation EditExerciseComment{   
       editExerciseComment( id: 3, content: "I edited this comment")
             { id 
              createdAt
              exerciseId
              parentId
              content   } 
}
```

If successful, there should be a result similar to the following:
![editted exercise comment](https://user-images.githubusercontent.com/77421872/215618227-7f3dbec4-8522-4446-bdad-918edc7276a0.png)
